### ISSUE TITLE
Resnet why "init_cls"=="increment"?

### ISSUE THREAD

**Issue by LightingMc** [G-U-N/PyCIL#80]
why is it designed like this? Why different kernel sizes?

PyCIL/convs/resnet.py

            if args["init_cls"] == args["increment"]:
                self.conv1 = nn.Sequential(
                    nn.Conv2d(3, self.inplanes, kernel_size=7, stride=2, padding=3, bias=False),
                    nn.BatchNorm2d(self.inplanes),
                    nn.ReLU(inplace=True),
                    nn.MaxPool2d(kernel_size=3, stride=2, padding=1),
                )
            else:
                self.conv1 = nn.Sequential(
                    nn.Conv2d(3, self.inplanes, kernel_size=3, stride=1, padding=1, bias=False),
                    nn.BatchNorm2d(self.inplanes),
                    nn.ReLU(inplace=True),
                    nn.MaxPool2d(kernel_size=3, stride=2, padding=1),
                )

**Comment by G-U-N** [comment]
Hi @LightingMc ,

Thank you for your interest.

This design is borrowed from [DER](https://github.com/Rhyssiyan/DER-ClassIL.pytorch/blob/main/inclearn/convnet/resnet.py#L115) and PodNet. Please see this [line](https://github.com/Rhyssiyan/DER-ClassIL.pytorch/blob/main/inclearn/convnet/resnet.py#L115).

In addition, could you be more polite for proposing issues? **Using "???" and "them" is  very offensive**. We are volunteers for maintaining the project and solving the issues.

**Comment by LightingMc** [comment]
Oh, I am sorry. Appreciate your help and effort :)
